File: src/main/java/uk/co/threebugs/mochiwhattotrade3/account/AccountController.java
package uk.co.threebugs.mochiwhattotrade3.account;

import lombok.RequiredArgsConstructor;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

import java.util.List;

@RequiredArgsConstructor
@RestController
public class AccountController {

    private final AccountService accountService;

    @GetMapping("/accounts")
    public List<AccountDto> findAll() {
        return accountService.findAll();
    }
}

File: src/main/kotlin/uk/co/threebugs/mochiwhattotrade3/account/AccountController.kt
package uk.co.threebugs.mochiwhattotrade3.account

import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController

@RestController
class AccountController(private val accountService: AccountService) {

        @GetMapping("/accounts")
    fun findAll(): List<AccountDto> {
        return accountService.findAll()
    }
}
